content-src/components/SiteIcon/SiteIcon.js

```diff
@@ -0,0 +1,95 @@
+const React = require("react");
+const classNames = require("classnames");
+
+const DEFAULT_FALLBACK_BG_COLOR = [200, 200, 200];
+const MIN_RESOLUTION = 2;
+const FAVICON_SIZE = 16;
+
+function getSiteImage(site, width) {
```

Comment:
Nit: the naming of this is a bit ambiguous, as you don't get back an image but a URL... maybe `getSiteImageURL()`.